src/main/java/org/apache/maven/plugins/clean/CleanMojo.java

```diff
@@ -197,7 +199,7 @@ public class CleanMojo extends AbstractMojo {
      * waiting for all files to be deleted when the session ends, <code>at-end</code> to indicate that the actual
      * deletion should be performed synchronously when the session ends, or <code>defer</code> to specify that
      * the actual file deletion should be started in the background when the session ends (this should only be used
-     * when maven is embedded in a long running process).
+     * when maven is embedded in a long-running process).
```

Comment:
period inside parentheses

Reply:
I see your point. Maybe even drop the parentheses altogether?